fetch_gazebo_demo/scripts/pick_place_demo.py

```diff
@@ -151,7 +151,7 @@ def updateScene(self):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         wait = False)
+                                         use_service = False)
```

Comment:
```suggestion
                                         )
```
@mikeferguson's suggestion to **use_service = True** makes sense to me.

GitHub suggestions don't support two line edits :disappointed: